env0/resource_module.go

```diff
@@ -12,12 +12,6 @@ import (
 )
 
 func resourceModule() *schema.Resource {
-	vcsExactlyOneOf := []string{
```

Comment:
using conflictsWith instrad.
Due to this comment: 
`when isGitLabEnterprise is true the token_id and token_name should not be required`